sci-libs/torchvision/torchvision-0.17.1.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 2020-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_SINGLE_IMPL=1
+DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_EXT=1
+inherit distutils-r1 multiprocessing
+
+DESCRIPTION="Datasets, transforms and models to specific to computer vision"
+HOMEPAGE="https://github.com/pytorch/vision"
+SRC_URI="https://github.com/pytorch/vision/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/vision-${PV}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	$(python_gen_cond_dep '
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/typing-extensions[${PYTHON_USEDEP}]
+		dev-python/pillow[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+		dev-python/scipy[${PYTHON_USEDEP}]
+	')
+	sci-libs/pytorch[${PYTHON_SINGLE_USEDEP}]
+	media-video/ffmpeg
+	dev-qt/qtcore:5
```

Comment:
Is it also compatible with qt6?

Reply:
I unmerged dev-qt/qtcore and torchvision built and ran without issue. So I suppose qt is not a necessary dependency. Not sure why it is introduced in previous version.

@AndrewAmmerlaan In this repo I saw all previous torchvision commits are authorized by you. Do you have any clue on why qt is introduced?

Reply:
I honestly can't remember, looking through the history it seems it was always there. But perhaps it is not needed. Is the size of the install tree the same when you compile with qtcore on the system versus without?

Reply:
>  Is the size of the install tree the same when you compile with qtcore on the system versus without

Yes, exactly the same.

Reply:
Then it is probably unnecessary indeed